llarp/path/path.hpp

```diff
@@ -394,6 +403,8 @@ namespace llarp
       uint64_t m_ExitObtainTX            = 0;
       PathStatus _status;
       PathRole _role;
+      util::DecayingHashSet< TunnelNonce > m_UpstreamBloomFilter;
+      util::DecayingHashSet< TunnelNonce > m_DownstreamBloomFilter;
```

Comment:
Why are we calling these bloom filters if they're really just hash maps?

Reply:
updated variable names